baseline/baseline.py

```diff
@@ -0,0 +1,145 @@
+# -*- coding:utf-8 -*-
+
+import json
+import sys
+sys.path.append('..')
+import mgnn.config_train as args
+import paths
+import re
+import csv
+import pickle
+import numpy as np
+from nltk.tokenize import word_tokenize
+from pretreatment.DataExtract import EntityLinking, GetPredicateList, Entity_Link_Falcon
+from pretreatment.QueryFilter import *
+from torchnlp.word_to_vector import FastText, GloVe
+fasttext = FastText()
+from embeddings import GloveEmbedding
+g = GloveEmbedding('common_crawl_840', d_emb=300)
+import math
+
+def get_ngram(text, n):
```

Comment:
Could you use this implementation instead. It returns both 1 and 2 word embeddings for the words in the sentence. For multi words the embeddings are averaged over.


```
def get_ngram_embeddings(text, n):
  embeddings = []
  for i in range(len(text)):
    for j in range(1, n + 1):
      words = text[i: i + j]
      if len(words > 0):
        embedding = [] # Please test that this line works as expected I was not able to exhaustively run this code
        for word in words:     
          embedding += fasttext[word].numpy()
        embeddings.append(embedding / len(words))
  return embeddings
```

Reply:
Got it! I'll replace that